pkg/minikube/bootstrapper/images/images.go

```diff
@@ -120,8 +121,8 @@ func etcd(v semver.Version, mirror string) string {
 }
 
 // archTag returns a CPU architecture suffix for images
-func archTag(hasTag bool) string {
-	if runtime.GOARCH == "amd64" && !hasTag {
+func archTag(needsArchSuffix bool) string {
```

Comment:
how about adding unittest for this func to be more clear what we need?

Reply:
done